### PR TITLE
les: log disconnect reason when light server is not synced

### PR DESCRIPTION
**Pull request by adamschmideg**
When the light server disconnects due to not being synced, it gives no reason for that. Provide that reason in a debug log message.

### CODE REVIEW

File: les/server_handler.go [hunk 1]
```diff
@@ -128,6 +128,7 @@ func (h *serverHandler) handle(p *peer) error {
 	}
 	// Reject light clients if server is not synced.
 	if !h.synced() {
+		p.Log().Debug("Not synced Light Ethereum rejecting", p)
```
Comment (ligi) [inline les/server_handler.go]: somehow this sentence does not feel like proper english
what about "rejecting light client that is not synced"

Comment (adamschmideg) [reply les/server_handler.go]: It's the server that's not synced. I agree it doesn't sound proper English, though.

Comment (adamschmideg) [reply les/server_handler.go]: How about "Light server not synced, so it's rejecting client ..."?

File: les/server_handler.go [hunk 2]
```diff
@@ -128,6 +128,7 @@ func (h *serverHandler) handle(p *peer) error {
 	}
 	// Reject light clients if server is not synced.
 	if !h.synced() {
+		p.Log().Debug("Light server not synced, so it's rejecting client", p)
```
Comment (ligi) [inline les/server_handler.go]: "Light server not synced, so it's rejecting client"
```suggestion
		p.Log().Debug("Light server not synced, so it's rejecting the client", p)
```

File: les/server_handler.go (hunk 1 above)
Comment (zsfelfoldi) [reply les/server_handler.go]: "Light server not synced, rejecting client" sounds fine to me

**Pull request merged**